app/src/main/java/org/simple/clinic/summary/teleconsultation/api/TeleconsultationsResponse.kt

```diff
@@ -5,6 +5,12 @@ import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class TeleconsultationsResponse(
+    @Deprecated(
+        message = "Use teleconsultationPhoneNumbers to get list of all MO phone numbers",
+        replaceWith = ReplaceWith("teleconsultationPhoneNumbers", "org.simple.clinic.summary.teleconsultation.api")
```

Comment:
This is incorrect usage of `ReplaceWith` since it should not cause compilation errors/failing tests after running. If we don't have a replacement that does not cause a compilation error, we should avoid using it and let the developer decide how to migrate a deprecated usage.

Reply:
Ah, okay I used it to make easier to replace at call sites. I will remove it.